src/app/containers/app/app.component.ts

```diff
@@ -315,7 +304,11 @@ export class AppComponent implements OnInit, OnDestroy {
       idleServiceName: 'idleSession'
     };
 
+    this.timeoutNotificationsService.notificationOnChange().subscribe(event => {
```

Comment:
would a take(1) work here so we can unsubscribe as we are finished with the event/subscription?

Reply:
No it wouldn't work in this case, I did try that